### ISSUE TITLE
missing markers?

### ISSUE THREAD

**Issue by eljojo** [findoutwho/hasbeen.in#193]
Hello friends!
I just pushed an update which caused quite a few markers to disappear from my profile page.

![screen shot 2017-10-24 at 11 20 56](https://user-images.githubusercontent.com/491230/31952025-9a7c8d84-b8ad-11e7-9939-879a902a63b0.png)

once I have time I'll debug myself but thought of sharing this in case you already know what caused it.

**Comment by bascht** [comment]
Oh noez. This could be a regression of #80 (or #59 / #62). 😭

**Comment by bascht** [comment]
![screenshot_20171026_161638](https://user-images.githubusercontent.com/51264/32057965-0e886be6-ba69-11e7-9896-233608697b88.png)
They're here. Yup. Weird. 🕵️

**Comment by eljojo** [comment]
I still fill that there some missing. (captured with adblock disabled)
<img width="617" alt="screen shot 2017-10-29 at 11 47 52" src="https://user-images.githubusercontent.com/491230/32145436-1abd261c-bc9f-11e7-8c73-a3e391ca05c4.png">

**Comment by bascht** [comment]
Yup, that's why I think it might be a regression of #59. The number of lookups per request is limited. That's why it takes some time until the map is full again. This is as of today:

![screenshot_20171102_105911](https://user-images.githubusercontent.com/51264/32320083-eab5c44e-bfbc-11e7-93a7-7ab634d81749.png)

**Comment by annismckenzie** [comment]
Can we switch to something like OpenStreetMap? 😬

**Comment by bascht** [comment]
The problem isn't the map rendering but the GeoCoder. See #59.

**Comment by annismckenzie** [comment]
I got that – I meant their Geocoder (http://wiki.openstreetmap.org/wiki/Nominatim).

**Comment by bitboxer** [comment]
It is worth a try. Hopefully they got better in the last couple of years 👍
